run_tests/README.md

```diff
@@ -0,0 +1,49 @@
+# Automated Testing Script
+
+This folder contains a script that will run through the tests in this repo using an automated UI integration testing approach. Because this uses a real browser with real hardware, it must be run on a machine with bluetooth functionality and a bluetooth device within range.
```

Comment:
```suggestion
This folder contains a script that will run through the tests in this repo using an automated UI integration testing approach. Because this uses a real browser with real hardware, it must be run on a machine with Bluetooth functionality and a bluetooth device within range.
```